db_sql.go

```diff
@@ -780,6 +792,15 @@ func createMigrations() []migration.Migrator {
 
 		// 8. We add AuthUserType field to the userstore, to determine what type of user account this is.
 		`ALTER TABLE authuserstore ADD COLUMN authusertype SMALLINT default 0;`,
+
+		// 9. Additional data fields as well as fields to keep track of changes to users
+		`ALTER TABLE authuserstore 
+			ADD COLUMN phone VARCHAR, 
+			ADD COLUMN remarks VARCHAR,
+			ADD COLUMN created BIGINT,
```

Comment:
I think rather use TIMESTAMP fields for time. The only thing you need to be careful about then, is to make sure you convert all times to UTC before inserting them into the DB.